docs/configuration.rst

```diff
@@ -125,3 +125,73 @@ Fake Gateway
 .. _`Twilio client`: https://pypi.python.org/pypi/twilio
 .. _python-qrcode: https://pypi.python.org/pypi/qrcode
 .. _`the upstream ticket`: https://code.google.com/p/google-authenticator/issues/detail?id=327
+
+Remember Browser
+----------------
+
+During a successful login with a token, the user may choose to remember this browser.
+If the same user logs in again on the same browser, a token will not be requested, as the browser
+serves as a second factor.
+
+The option to remember a browser is deactived by default. Set `TWO_FACTOR_REMEMBER_COOKIE_AGE` to activate.
+
+The browser will be remembered as long as
+- the cookie that authorizes the browser did not expire,
+- the user did not reset his password, and
```

Comment:
Custom users might not use `password` to store their password, .e.g [django-auth-ldap](https://django-auth-ldap.readthedocs.io/en/latest/users.html#user-objects)

As far as I can see, the code will work just fine but we probably should warn users if they use a custom user then remember tokens might not get reset.

Reply:
Also that should be "their" not "his"